src/Symfony/Component/HttpFoundation/StreamedJsonResponse.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation;
+
+use function array_keys;
+use function explode;
+use function flush;
+
+use Generator;
+
+use function json_encode;
+
+use const JSON_THROW_ON_ERROR;
```

Comment:
Symfony CS implies not adding `use` for the global namespace

Reply:
fixed